android/quest/src/test/java/org/smartregister/fhircore/quest/util/mappers/MeasureReportPatientViewDataMapperTest.kt

```diff
@@ -52,7 +52,10 @@ class MeasureReportPatientViewDataMapperTest : RobolectricTest() {
     with(profileViewDataHiv) {
       Assert.assertEquals("TEST_PATIENT", logicalId)
       Assert.assertEquals("Bareera Hadi", name)
-      Assert.assertEquals("23y", age)
+      Assert.assertEquals(
+        "24y",
```

Comment:
Since we know the actual DOB from the [test data JSON](https://github.com/opensrp/fhircore/blob/main/android/quest/src/test/resources/patient-registration-questionnaire/sample/patient.json) i.e. **1998-12-14** we can make the expected value dynamic by doing something like this

```
LocalDate dob = LocalDate.of(1998, 12, 14);  
Period period = Period.between(dob, LocalDate.now());  
expectedDate = "${period.getYears()}y"
```

Reply:
Yeah,  it makes sense to compute the value of the age from within the test 👍🏾